lib/dealership.rb

```diff
@@ -0,0 +1,53 @@
+class Dealership
+  attr_reader :name, :address, :inventory, :car
```

Comment:
Careful with putting `:car` in your attr_reader.... Is `:car` really an attribute of Dealership?

Reply:
(Trick question, no it is not because you don't use `@car` anywhere in your code) ;)